src/ontology/mondo-edit.obo

```diff
@@ -249108,6 +249120,7 @@ def: "Autosomal recessive limb-girdle muscular dystrophy type 2M (LGMD2M) is a f
 subset: ordo_disease {source="Orphanet:206554"}
 synonym: "autosomal recessive limb-girdle muscular dystrophy caused by mutation in FKTN" EXACT [MONDO:design_pattern]
 synonym: "FKTN autosomal recessive limb-girdle muscular dystrophy" EXACT [MONDO:design_pattern, MONDO:patterns/disease_series_by_gene]
+synonym: "LGMD-FKTN related" EXACT CLINGEN_PREFERRED []
```

Comment:
could you please add a dbxref here:
https://clinicalgenome.org/affiliation/50061/